uint32 frameNumber
byte subsampleNumber
float64 fx_N
float64 fy_N
float64 fz_N
float64 mx_Nm
float64 my_Nm
float64 mz_Nm
